plugins/Live/lang/en.json

```
@@ -1,69 +1,69 @@
+        "QueryMaxExecutionTimeExceeded": "Could not run the query in time.",
+        "QueryMaxExecutionTimeExceededReasonDateRange": "Maybe the selected date range is too wide?",
+        "QueryMaxExecutionTimeExceededReasonSegment": "Maybe the selected segment didn't match any visits?",
+        "QueryMaxExecutionTimeExceededReasonUnknown": "Please try again. Get in touch with your administrator or support if the problem persists.",
         "VisitorLog": "Visits Log",
-        "VisitorLogDocumentation": "This table shows the latest visits within the selected date range. You can see when a visitor's last visit occurred by hovering over the date of a visit. %1$s If the date range includes today, you can see your visitors real time! %2$s The data displayed here is always live, regardless of whether and how often you are using the archiving cron job.",
-        "VisitorLogNoDataMessagePurged": "Likely the data has been purged because the regular deletion of old raw data is enabled and the date for this report is more than %s days old. A super user can change this setting by going to Administration => Privacy.",
+        "VisitorLogDocumentation": "This table shows the latest visits within the selected date range. Hover over the date of a vist to find out when the most recent one occured. %1$s If the date range includes today, you can see your visitors in real-time. %2$s The data displayed here is always live, regardless of whether and how often you make use of the archiving cron-job.",
+        "VisitorLogNoDataMessagePurged": "The data has likely been purged because the regular deletion of old raw data is on, and the date for this report is more than %s days old. A superuser can change this setting by going to Administration → Privacy.",
         "VisitorProfile": "Visitor profile",
-        "VisitorsInRealTime": "Visits in Real-time",
-        "VisitorsLastVisit": "This visitor's last visit was %s days ago.",
+        "VisitorsInRealTime": "Visits in real-time",
+        "VisitorsLastVisit": "Also visited %s days earlier.",
         "VisitsFrom": "%1$s%2$s visits%3$s from",
         "VisitSummary": "Spent a total of %1$s%2$s on the website%3$s, and viewed %4$s pages%5$s in %6$s visits%7$s.",
         "VisitSummaryWithActionDetails": "Spent a total of %1$s%2$s on the website%3$s, and performed %4$s actions%5$s (%6$s) in %7$s visits%8$s.",
-        "RowActionTooltipDefault": "Show Visits Log segmented by this row",
-        "RowActionTooltipWithDimension": "Show Visits Log segmented by this %s",
-        "RowActionTooltipTitle": "Open segmented Visits Log",
-        "SegmentedVisitorLogTitle": "Visits Log showing visits where %1$s is \"%2$s\"",
-        "OnClickPause": "%s is started. Click to pause.",
-        "OnClickStart": "%s is stopped. Click to start.",
+        "RowActionTooltipDefault": "Show the visits log for each segment of this row",
+        "RowActionTooltipWithDimension": "Show %s visits by each of its subcategories.",
```

Those still don't seem to have the same meaning as before. Both are tooltips for an icon, that (on click) opens the visits log, while directly applying a segment to the visits log that is based on the row the icon was clicked in. That's what `segment by this row` should say. Maybe we can keep the old wording? Or what doesn't suite you with it?

---

```suggestion
        "RowActionTooltipDefault": "Show the visits log for each segment of this row",
        "RowActionTooltipWithDimension": "Show the visits log segmented by this %s",
```
I misunderstood.
Was hoping to avoid "segmented by", but I guess this is OK enough.